src/app/modules/core/components/map/map.component.ts

```diff
@@ -160,7 +161,7 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit, OnC
     this.MapService.reportlayerGroup.subscribe(reportlayerGroup => {
       this.reportlayerGroup = reportlayerGroup;
     });
-    //#endregion
+    var scale = L.control.scale();
```

Comment:
I think this can be removed? The scale gets added to the map on line 200, right?

Reply:
Yes, this was a previous attempt that did not work. Good catch. Line 164 removed.